hibernate-core/src/test/java/org/hibernate/orm/test/annotations/onetoone/OneToOneJoinTableUniquenessTest.java

```diff
@@ -44,7 +44,7 @@ protected Class<?>[] getAnnotatedClasses() {
 	protected void configure(Configuration configuration) {
 
 		try {
-			output = File.createTempFile( "update_script", ".sql" );
+			output = Files.createTempFile( "update_script", ".sql" ).toFile();
```

Comment:
I don't get it. How is this an improvement?

`File.createTempFile()` is shorter and isn't deprecated.

Reply:
I mean, I guess we really aren't interested in running any bot over our whole code base if it's going to be making seemingly-unnecessary or "opinionated" changes like this.

Reply:
Lots of Picnic's checks are opinionated, so I would then recommend against merging this PR.

That said, the specific replacement suggested here relates to [this security issue](https://rules.sonarsource.com/java/type/Security%20Hotspot/RSPEC-5443/). Those docs may indicate that further changes are necessary, but (for Linux at least) [my tests](https://github.com/PicnicSupermarket/error-prone-support/pull/1162#discussion_r1584176411) indicated that this change suffices.

Reply:
Thanks, @Stephan202 

To be clear, these uses of `Files.createTempFile` are in **tests**. 

So there's definitely no security issue here.